blocks/leadspace/leadspace.css

```diff
@@ -1,4 +1,36 @@
 /* Leadspace block styles */
+.leadspace, .leadspace p {
+    font-family: var(--sans-serif-font-light);
```

Comment:
ISn't this inherited from body?

Reply:
Both `.leadspace` and `.leadspace p` are needed?

Reply:
Yes, unfortunately both are needed because of the Franklin way of stripping off the P when there is only 1 paragraph in a cell. (I learned this yesterday).

The body has a different spec, I think it is font size 21 (not sure about the rest) so Amol wanted me to level set fonts specifically for Leadspace.

Reply:
No, I think it picked up the body P (which was 21px size).

Reply:
![image](https://user-images.githubusercontent.com/3883795/228945209-161adf10-e8a8-43d9-b5c6-fa01598b00a6.png)

there was no P style coming through, which is why Amol had me add these styles as a level-set.